modules/exploits/windows/smb/ms17_010_eternalblue.rb

```diff
@@ -187,6 +187,10 @@ def smb_eternalblue(process_name, grooms)
       print_status("Receiving response from exploit packet")
       code, raw = smb1_get_response(sock)
 
+      if code.nil?
+        print_error("Did not receive a response from exploit packet")
```

Comment:
Not sure. It didn't seem to hurt to let it keep going there (if its patched, its still patched). It may take up some time, but thats about it.